### PR TITLE
Fix unit test build on Windows

### LINKED ISSUE #2630: making package testingutil and ui work for unit test build on windows
/kind bug

## What versions of software are you using?

**Operating System:** windows

**Output of `odo version`:** master

## How did you run odo exactly?

```make test```
Note: odo binary is not needed for this test

## Actual behavior

Unit test build gets failed on windows for packages mentioned in the issue title.
More details : https://github.com/openshift/odo/pull/2562#issuecomment-586195763

## Expected behavior

Unit test should pass. 

It could be a proposed solution : https://github.com/tektoncd/cli/pull/191

### PR DESCRIPTION
**Pull request by kadel**
in glide.yaml replace github.com/kr/pty with github.com/creack/pty

**What type of PR is this?**

/kind bug

**What does does this PR do / why we need it**:

Successful UT build on windows 

**Which issue(s) this PR fixes**:

Fixes #2630 

**How to test changes / Special notes to the reviewer**:
Run ```make test``` on windows, It should run successfully.

Credit goes to @prietyc123, she was the one who figured out that we need to use  github.com/creack/pty instead of github.com/kr/pty

This PR replaces https://github.com/openshift/odo/pull/2766

### CODE REVIEW

File: glide.yaml [hunk 1]
```diff
@@ -27,6 +27,9 @@ import:
 - package: k8s.io/api
   version: origin-4.1-kubernetes-1.13.4
   repo: https://github.com/openshift/kubernetes-api.git
+- package: github.com/kr/pty
+  version: v1.1.9
+  repo: https://github.com/creack/pty
```
Comment (girishramnani) [inline glide.yaml]: Sneaky ✌️

**Comment by kadel** [comment]
ping @girishramnani @cdrage

**Pull request merged**